08-interop/08-interop.md

```diff
@@ -169,3 +169,109 @@ Task #10 - UDP between lwIP and emb6 on iotlab-m3
 Link-local UDP over IPv6 packet exchange (payload length 8) between an iotlab-m3
 node running RIOT with lwIP and an iotlab-m3 node running RIOT with emb6 (in
 both directions).
+
+Task #11 - UDP exchange between iotlab-m3 and Zephyr
+=====================================================
+### Description
+
+UDP packet exchange between an iotlab-m3 node running RIOT and a samr21-xpro
+node running Zephyr.
+
+### Testing procedure
+
+#### Setting up the Zephyr node
+
+1. Follow the
+['Getting Started'](https://docs.zephyrproject.org/latest/getting_started/index.html)
+guide on Zephyr's documentation.
+
```

Comment:
Newline not needed (in fact it unnecessarily spreads out the numbered list.

Reply:
Removed newlines